service/Microsoft.Azure.Devices/AmqpServiceClient.cs

```diff
@@ -101,8 +101,10 @@ public SendingAmqpLink SendingLink
 
         public async override Task CloseAsync()
         {
+            this.faultTolerantSendingLink.Dispose();
```

Comment:
Yes.

---
In reply to: [103077496](https://github.com/Azure/azure-iot-sdk-csharp/pull/57#discussion_r103077496) [](ancestors = 103077496)